### PR TITLE
Add transfer Issue plugin to Kubeflow repos

### PR DESCRIPTION
**Pull request by andreyvelich**
I added cherrypicker and transfer issue plugins to the Kubeflow repos:
-  https://prow.k8s.io/command-help#cherrypick 
- https://prow.k8s.io/command-help#transfer_issue

This should help us with Kubeflow repo migration: https://github.com/kubeflow/kubeflow/issues/7549

Also, added `kubeflow/dashboard` repo to the Kubeflow list.

/hold for review

/cc @rimolive @terrytangyuan @kimwnasptd @thesuperzapper @StefanoFioravanzo @zijianjoy @johnugeorge @jbottum @james-jwu

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]
/lgtm

**Comment by andreyvelich** [comment]
Sure.
/assign @chaodaiG @chases2 @cjwagner @fejta @michelle192837 @mpherman2

**Comment by andreyvelich** [comment]
>In cases where things are very ambiguous (technical discussions covering multiple components)

Why we can't use `kubeflow/manifests` for Kubeflow platform questions (cc @juliusvonkohout) or `kubeflow/community` for the new project adoption and questions for the community direction?
We can always reconsider it in the future, if that won't work for us.

I feel that it will be really hard to triage issues under `kubeflow/kubeflow` for community members and our users will feel bad about it.

**Comment by andreyvelich** [comment]
Additionally, I added `trigger` plugin to the Kubeflow: https://prow.k8s.io/command-help#ok_to_test.
That will allow us to add `/ok-to-test` label to the PRs and trigger GitHub actions with `/rerun-all` command. Currently, you have to have write access to add this label to the PR.

We already use this GitHub action in several repos (Pipelines, Katib, Training Operator) to re-trigger GitHub actions using PR comments: https://github.com/estroz/rerun-actions.
This actions expect `ok-to-test` label to be present.

cc @rimolive

**Comment by rimolive** [comment]
I'm good with the changes.

**Comment by andreyvelich** [comment]
@thesuperzapper Do you know if I setup the cherrypicker plugin correctly ?
I just follow the same config as for Kubernetes prow: https://github.com/kubernetes/test-infra/blob/ab0aea453361484ad834f42c7cb633f5f74a3535/config/prow/plugins.yaml#L1826.

**Comment by andreyvelich** [comment]
@rimolive @terrytangyuan @kimwnasptd @thesuperzapper @StefanoFioravanzo @zijianjoy @johnugeorge @jbottum @james-jwu Anything you want to add before merging this PR ?

**Comment by rimolive** [comment]
No other things to add. Thanks @andreyvelich for working in this PR.

**Comment by andreyvelich** [comment]
@zijianjoy @chensun @james-jwu  @chaodaiG @chases2 @cjwagner @fejta @michelle192837 @mpherman2 Please can you help us to LGTM this PR ?

File: prow/oss/plugins.yaml
Comment (cjwagner) [inline prow/oss/plugins.yaml:407]: There is no cherrypicker service deployed on this Prow instance so there won't be anything to reach at this endpoint.

**Review by cjwagner — CHANGES_REQUESTED** [review]
I'm not sure how much work it would take to set up the cherrypicker external-plugin for this Prow instance. If that is something you would like can you please open a separate GitHub issue requesting that?

If you remove the external_plugins section for now I can approve the rest of this PR though.

File: prow/oss/plugins.yaml
Comment (andreyvelich) [reply prow/oss/plugins.yaml:407]: I see, thanks for checking @cjwagner! I will create separate issue and remove it from this PR.

**Review by cjwagner — APPROVED** [review]

File: prow/oss/plugins.yaml
Comment (juliusvonkohout) [reply prow/oss/plugins.yaml]: I tested it with https://github.com/kubeflow/manifests/issues/2837#issuecomment-2277454689 and it worked out fine.

Comment (andreyvelich) [reply prow/oss/plugins.yaml]: Amazing, thank you @juliusvonkohout!

File: prow/oss/plugins.yaml
Comment (hbelmiro) [inline prow/oss/plugins.yaml:345]: @andreyvelich if my understanding is correct, `/ok-to-test` should trigger GitHub Actions after [this change](https://github.com/GoogleCloudPlatform/oss-test-infra/pull/2331/files#diff-61d4ecf8ed653ce1a03b8a3758c9fa61af90500fd05f6580dbafcfef3ec780f0R345). But it didn't.
Screenshot from 2024-08-13 12-57-51

https://github.com/kubeflow/pipelines/pull/11096#issuecomment-2286585861

Do you know if there's anything else that we should do, or is that a bug?

cc @rimolive @DharmitD

Comment (andreyvelich) [reply prow/oss/plugins.yaml:345]: @hbelmiro This plugin just adds `ok-to-test` label to the PR after we add this comment: `/ok-to-test`.
It doesn't trigger to GitHub actions.
Since we use this action to trigger our action: https://github.com/estroz/rerun-actions, you need to add this comment to trigger the actions:
```
/rerun-all
```

Comment (hbelmiro) [reply prow/oss/plugins.yaml:345]: I see.
The comment made me think that `/ok-to-test` would somehow automatically trigger `/rerun-all`.
Thank you @andreyvelich.

**Pull request merged**
